android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/GetStartedActivity.java

```diff
@@ -128,10 +129,37 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
           startActivity(i);
           overridePendingTransition(android.R.anim.fade_in, R.anim.hold);
         }
+
+        uncheckGetStartedIfComplete();
       }
     });
   }
 
+  /**
+   * Uncheck show "Get Started" on startup if:
+   * User hasn't set a preference on showing "Get Started",
+   * Keyman enabled as a system-wide keyboard, and
+   * Keyman set as default keyboard
+   */
+  private void uncheckGetStartedIfComplete() {
+    if (SystemIMESettings.isEnabledAsSystemKB(this) &&
+        SystemIMESettings.isDefaultKB(this)) {
+
+      final SharedPreferences prefs = getSharedPreferences(getString(R.string.kma_prefs_name), Context.MODE_PRIVATE);
+      if (!prefs.contains(showGetStartedKey)) {
+        // Everything is completed, so un-check "Get Started" on startup.
+        // onCheckedChanged() will save the preference
+        final CheckBox checkBox = (CheckBox) findViewById(R.id.checkBox);
+        checkBox.setChecked(false);
+
+        // Save preference
+        //SharedPreferences.Editor editor = prefs.edit();
+        //editor.putBoolean(showGetStartedKey, false);
+        //editor.commit();
```

Comment:
```suggestion
```

Reply:
Good catch! I commented it out after I saw `onCheckedChanged()` handled it.